app/src/main/java/co/ostorlab/insecure_app/bugs/calls/HardcodedUrlInUrl.java

```diff
@@ -0,0 +1,27 @@
+package co.ostorlab.insecure_app.bugs.calls;
+import co.ostorlab.insecure_app.BugRule;
+
+
+import android.annotation.SuppressLint;
+import android.util.Log;
+
+
+public final class HardcodedUrlInUrl extends BugRule {
+
+    private static final String TAG = "RULE";
+
+    @SuppressLint("AuthLeak")
+    public String get_url() {
+        return "https://ostora:07b8a0abfx53p98f7ae5@ostora.ostorlab.com/faq/?country=morocco";
+    }
+
+    @Override
+    public void run() throws Exception {
+        Log.i(TAG, String.format("Message: %s", get_url()));
```

Comment:
Can you please add code that open that URL in a webview.

Reply:
yes of course